src/Microsoft.SqlTools.ServiceLayer/Metadata/Contracts/ObjectMetadata.cs

```diff
@@ -30,6 +30,8 @@ public class ObjectMetadata
         public string Schema { get; set; }
```

Comment:
What is the MetadataType set to for things like triggers that aren't in the MetadataType enum?